registry/kusama.json

```diff
@@ -78,5 +78,13 @@
         "shortDescription": "DotSama native node operator and community supporter",
         "longDescription": "## About Sik\nFounder of [Crifferent Staking Services](https://crifferent.de) \n\nDeveloper for [Staking Facilities](https://stakingfacilities.com) \n\nMember of [DOT Validator Alliance DAO](https://dotvalidators.org) \n\nTeam member @[HydraDX](https://hydradx.io)/[Basilisk](https://bsx.fi) \n\nRPC provider @[UnitedBloc](https://unitedbloc.com/) \n\nMember of the [Infrastructure Builders Program](https://ibp.network/)\n\n---\n\n**From within the ecosystem - for the ecosystem!** \n\nI'm evaluating proposals by taking different views into consideration. I'm not only looking at it from the perspective of a community member, but also as a developer/founder/infra-provider/human being. \n\n---\n## Contact me \n\n@dev0_sik @[Twitter](https://twitter.com/dev0_sik) \n\n@dev0_sik @[Telegram](https://t.me/dev0_sik) \n\nsik#3706 @[Discord](https://discord.com/) \n\n@dev0_sik:matrix.org @[Element](https://element.io/) \n\n[simon.kraus@crifferent.de](mailto:simon.kraus@crifferent.de) \n\nWhatsApp/Callboy +49 157 377 309 02",
         "isOrganization": false
+     },
+    {
+        "address": "HBtJ7Poz9ya9pzS8AXK16tAqJ1qxXbVspkM9E6p57BJ1hJs",
+        "name": "Abdulbee",
+        "image": "https://raw.githubusercontent.com/nova-wallet/opengov-delegate-registry/master/images/Abdulbee_Nova.jpg",
+        "shortDescription": "Polkadot Community Moderator, Anti-Scam executive team member, Head ambassador for content, a Rust afficionado",
+        "longDescription": "## About Abdulbee\nMy name is Abdulbasit (Abdulbee), a Rust afficionado/technical writer who currently contributes full-time in the ecosystem as a Polkadot head ambassador for content, an executive team member of the Polkadot anti-scam team, a community moderator of Polkadot’s official channels, and a QA officer at Dot.alert().\nI also currently curate two bounties (the anti-scam bounty and the community moderation team bounty).\nMy ecosystem philosophy is that good reputation (within or outside the ecosystem) is a good internal incentive for successful deliveries and that a key criteria to take into consideration when making governance decisions is the reputation of teams/individuals who submit proposals via governance. \nYou can reach me on discord (abdulbee#4681) or matrix (abdulbee:matrix.org). \nThank you!" 
+        "isOrganization": false
```

Comment:
in order to support our format type and fix some json problems, I suggest to use that type of formatting, how it will look like you can find in comment above.

```suggestion
    },
    {
        "address": "HBtJ7Poz9ya9pzS8AXK16tAqJ1qxXbVspkM9E6p57BJ1hJs",
        "name": "Abdulbee",
        "image": "https://raw.githubusercontent.com/nova-wallet/opengov-delegate-registry/master/images/Abdulbee_Nova.jpg",
        "shortDescription": "Polkadot Community Moderator, Anti-Scam executive team member, Head ambassador for content, a Rust afficionado",
        "longDescription": "## About Abdulbee\n\nMy name is Abdulbasit (Abdulbee), a Rust afficionado/technical writer who currently contributes full-time in the ecosystem as a Polkadot head ambassador for content, an executive team member of the Polkadot anti-scam team, a community moderator of Polkadot’s official channels, and a QA officer at Dot.alert().\n\nI also currently curate two bounties (the anti-scam bounty and the community moderation team bounty).\n\nMy ecosystem philosophy is that good reputation (within or outside the ecosystem) is a good internal incentive for successful deliveries and that a key criteria to take into consideration when making governance decisions is the reputation of teams/individuals who submit proposals via governance. \n\nYou can reach me on discord (abdulbee#4681) or matrix (abdulbee:matrix.org). \n\nThank you!",
        "isOrganization": false
```

Reply:
Okay, done.